src/components/WithdrawRow/index.tsx

```diff
@@ -23,8 +24,8 @@ interface WithdrawRowInterface {
 }
 
 const WithdrawRow: React.FC<WithdrawRowInterface> = ({ amount, balance, price, setAmount, error }) => {
-  const lbtcUnit = useSelector((state: any) => state.settings.denominationLBTC);
-  const currency = useSelector((state: any) => state.settings.currency.value);
+  const lbtcUnit = useSelector((state: RootState) => state.settings.denominationLBTC);
```

Comment:
You can also do that:
```
in some config file:
export type AppDispatch = typeof store.dispatch;
export type RootState = ReturnType<typeof store.getState>;
export const useTypedDispatch = (): AppDispatch => useDispatch<AppDispatch>();
export const useTypedSelector: TypedUseSelectorHook<RootState> = useSelector;

in component:
const currency = useTypedSelector(({ settings }) => settings.currency.value);
```

Reply:
Let's wait for [this](https://github.com/vulpemventures/marina/pull/237) to be merged (it introduces memoized selectors which removes the need to define a new UseSelector hook)

Reply:
🤔 think you confusing project/repos?

Reply:
Owh yeah sorry my bad, was Marina, anyway I think we should use `createSelector` instead of typing a custom useSelector